src/common/paths/tasks_taskID_runs_runID.yml

```diff
@@ -19,23 +23,61 @@ get:
       description: The run ID.
   responses:
     "200":
-      description: Returns the run record.
+      description: Success. The response body contains the task run.
       content:
         application/json:
           schema:
             $ref: "../schemas/Run.yml"
+        examples:
+          runSuccess:
```

Comment:
I think this all just needs indented one more stop then re-run `make generate` to fix the openapi spec errors